### DISCUSSION TITLE
Calling PostgreSQL UTC Now from Prisma method

### DISCUSSION THREAD

**Discussion by n-glaz** [prisma/prisma#15862] [Q&A]
I have an update method, and in that method I want to update a DateTime column value to now according to my PostgreSQL DB's clock, not my app server's clock. 

I can see how to do this on Models in the Schema, but I can't find a doc for doing it from my app, e.g. something like:

```typescript
prisma.resource.update(where: {...}, data: {completedAt: @db.now})
```

Guidance would be greatly appreciated!

**Comment by nurul3101** [comment]
Hello @n-glaz 👋 
Could you use `new Date().toISOString()` to set the UTC time?
There is also this GitHub [Discussion](https://github.com/prisma/prisma/discussions/5582) which discusses various solutions which should be helpful.
Also, we have a feature request: [#5598](https://github.com/prisma/prisma/issues/5598) to support `now()` as a value for setting DateTime outside of schema file, it would help our product team prioritise it if you could leave a 👍 to the request.

**Reply by vladi-stevanovic** [reply]
:wave: Hello @n-glaz  , I’m going through pending GitHub Discussions to check if you need any help. Is this issue still open?

Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! :raised_hands:

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
